de-lib/src/main/java/org/iplantc/de/diskResource/client/views/toolbar/DiskResourceViewToolbarImpl.java

```diff
@@ -530,6 +530,11 @@ void onNewPathListFileClicked(SelectionEvent<Item> event) {
         presenter.onCreateNewPathListSelected();
     }
 
+    @UiHandler("newMultiInputPathListMi")
+    void onNewMultiInputPathListFileClicked(SelectionEvent<Item> event) {
+        presenter.onCreateMultiInputPathListSelected();
```

Comment:
Instead of referencing the presenter, can we fire an event here?  In one of my DiskResources refactor PRs, I completely removed the presenter and converted everything to using events.